src/NodeMailer.purs

```diff
@@ -54,11 +53,11 @@ foreign import data Transporter :: Type
 
 foreign import data MessageInfo :: Type
 
-sendMail :: Message -> Transporter -> Aff Unit
-sendMail message transporter = void $ sendMail_ message transporter
+sendMail_ :: Transporter -> Message -> Aff Unit
+sendMail_ transporter message = void $ sendMail transporter message
 
-sendMail_ :: Message -> Transporter -> Aff MessageInfo
-sendMail_ message transporter = fromEffectFnAff $ runFn2 _sendMail (write message) transporter
+sendMail :: Transporter -> Message -> Aff MessageInfo
+sendMail transporter message = fromEffectFnAff $ runFn2 _sendMail transporter (unsafeToForeign message)
```

Comment:
> because usually xxx_ methods return Unit

Sorry, I've never heard of it.

Functions with `_` suffix that it returnes `Unit` exist surely, but also functions with `_` suffix that it doesn't returne `Unit` exist.
In short, I think that doesn't mean "xxx_ method have to return `Unit`".

> swap args because Transport is changing less often

Sorry, I can't understand purpose of this change.
The subject data is usually passed as last arg.
So If possible, I want you to explain the purpose objectively.

Reply:
> Functions with _ suffix that it returnes Unit exist surely

it's convention

> but also functions with _ suffix that it doesn't returne Unit exist

never seen

> The subject data is usually passed as last arg

yes, and `mail` is more subjectful than `transport`

```
transport <- 
mySend = sendMail transport

# than

mySend mail = sendMail mail transport

Reply:
If you say "it's convention", I think that you should show rationale, like "docs", "articles", "ratio of xxx_ that returns Unit and xxx_ that doesn't returns Unit on pursuit" and so on. Could you do that ?

---

The subject that I saied means like receiver object. In this case, it's `Transporter`.

I want you to check some packages.
https://pursuit.purescript.org/packages/purescript-arrays
https://pursuit.purescript.org/packages/purescript-tuples
https://pursuit.purescript.org/packages/purescript-web-dom
https://pursuit.purescript.org/packages/purescript-web-html

and so on. (there are same interface in too many packages)

Reply:
> Could you do that ?

there are no style guide for purescript that mention that

I's just something I have seen a lot in code of other people in purescript and haskell and I have never seen them use it in reverse (`_` to mark functions that DONT return Unit)

if I find - I will send

__________

probably you relate to `setAttribute :: String -> String -> Element -> Effect Unit`

ok, as you want

but I still think that `Transport` is being used often as `mySend = sendMail transport`, in case of `setAttribute` - this function is more one time and often used with all arguments simultaneously

but I dont want to discuss this anymore

Reply:
> I's just something I have seen a lot in code of other people in purescript and haskell and I have never seen them use it in reverse (_ to mark functions that DONT return Unit)

example: https://pursuit.purescript.org/packages/purescript-react-basic-dom/3.2.0/docs/React.Basic.DOM.SVG#v:set_

There are cases where Unit is not returned like this, but do you want to continue renaming?
If so, I'm willing to accept it.

---

> but I dont want to discuss this anymore

I got it. I don't say anymore, but i can't approve the change of args order, sorry.